### DISCUSSION TITLE
Stochastic sequence playback mode

### DISCUSSION THREAD

**Discussion by version-aotearoa** [SynthstromAudible/DelugeFirmware#3005] [Ideas]
Also known as Stagger/drunken walk, Brownian Motion algorithm or step action probability.

An amazingly generative (not reductive) playback mode which is available on some serious sequencers: Cirklon, Numerology/Vector, OxiOne etc.... and some not so serious sequencers e.g. Korg SQ-64, Pittsburgh Micro Sequence 😅
Basically the play head can move with probability: forward 50%, backward 25% or rest 25%
This creates a wandering phrase of your chosen sequence notes (think Autechre).
It will re-order or space out your notes - not randomise them, but rather controlled variation/iteration 😎

**Comment by worker-gray** [comment]
sounds crazy in a good way. gets my vote!

**Comment by version-aotearoa** [comment]
from Sound On Sound KORG SQ-64
“Stochastic mode, on the other hand, is completely fascinating. The sequencer has to make a decision as to whether the next step will be one forward, one backwards, two forward or the same step.
Every time you enable Stochastic mode the probability of each of those happening changes. Watching the white LEDs meander their way along the steps can be very absorbing, as are the unexpected melodies it generates.”
https://www.soundonsound.com/reviews/korg-sq-64

**Reply by seangoodvibes** [reply]
You'll be able to do this with the arp, doing this with the actually sequencer is a lot more difficult
